trino-connector/integration-test/trino-test-tools/trino-cascading-env/init/trino-remote/init.sh

```diff
@@ -26,13 +26,13 @@ cp "$trino_conf_dir/config/config.properties" /etc/trino/config.properties
 cp "$trino_conf_dir/config/jvm.config" /etc/trino/jvm.config
 cp "$trino_conf_dir/config/log4j2.properties" /etc/trino/log4j2.properties
 cp "$trino_conf_dir/config/catalog/gravitino.properties" /etc/trino/catalog/gravitino.properties
+cp /usr/lib/trino/plugin/mysql/mysql-connector-j-8.2.0.jar /usr/lib/trino/plugin/iceberg
 #
```

Comment:
Why don't you copy the pg connector

Reply:
The PG driver is already in iceberg folder, no need to copy.